dockerfiles/Dockerfile.dev

```
@@ -1,8 +1,12 @@
 ARG     BUILD_BASE
 
-FROM    ${BUILD_BASE}
+FROM    ${BUILD_BASE} as dev
 
-RUN     apk add -U git bash
+RUN     apk --no-chache add -U git
```

Sorry, my previous comment wasn't clear. I meant:

```suggestion
RUN     apk add --no-cache \
    bash \
    curl \
    git \
    make \
    mercurial
```